metabrainz/templates/index/team.html

```diff
@@ -0,0 +1,120 @@
+{% extends 'base.html' %}
+
+{% block title %}Team - MetaBrainz Foundation{% endblock %}
+
+{% block content %}
+<h1 class="page-title">Meet the MetaBrainz Foundation Team</h1>
+<div id="team" class="col-sm-12"> 
+ 
+  <div class="people">
+    <div class="pic long">
+      <img width="150px" height="150px" src="{{ url_for('static', filename='img/ruaok.jpg') }}">
+    </div>
+    <div>
+      <h4><strong>Executive Director</strong> - Robert Kaye | ruaok</h4>
+      <p>Robert is the founder and lead geek for <a href="https://musicbrainz.org/">MusicBrainz</a>, and Executive Director of the MetaBrainz Foundation, the non-profit organization that pays the bills for MusicBrainz. After studying Computer Engineering at Cal Poly, San Luis Obispo, he joined Xing Technology where he shaped the MP3 efforts and then spent time for the dot com boom on EMusic’s FreeAmp team. While at EMusic he started working on MusicBrainz, the open music encyclopedia, and fell in love with Open Source software.</p>
+      <p>Robert is also an active hardware hacker, creating art projects for Burning Man and Nowhere. He is the co-founder of <a href="http://partyrobotics.com">Party Robotics</a> where he co-created Bartendro the open source cocktail robot.</p>
+      <p>He lives in Barcelona, Spain. Follow Robert on Twitter: <a href="https://twitter.com/mayhembcn"> @mayhemBCN</a></p>
+    </div>
+  </div>
+  
+  <div class="people short">
+    <div class="pic">
+      <img width="150px" height="150px" src="{{ url_for('static', filename='img/bitmap.jpg') }}"/>
```

Comment:
I would create a subdirectory inside *img*, like *team*, move all images there, and name them like this: *\<first_name\>_\<last_name\>* (all lowercase).